lighthouse-core/lib/element.js

```diff
@@ -63,7 +63,8 @@ class Element {
           return null;
         }
         return this.driver.getObjectProperty(resp.object.objectId, propName);
-      });
+      })
+      .catch(() => null);
```

Comment:
FWIW I'm not sure we should fail silently here because the semantics end up being a bit weird (pretending the property doesn't exist when it's actually the element that doesn't). It seems like the caller should handle the exception (as it now does above).

Reply:
This is consistent with the current behavior of the method though, returning `null` when the object does not exist. If we keep throwing here, we should convert the above to throw and change the typedef

Reply:
Ah, good point, I missed the first check. I wonder if we ever actually fail here then. The issue seems like it's been the event listener gatherer that's been failing. All good regardless.